.gitignore

```diff
@@ -1,5 +1,5 @@
-# file generated by NetBeans IDE
 nb-configuration.xml
+nbactions.xml
```

Comment:
Deze lijken me niet meer relevant, dus dit kan weg volgens mij

Reply:
daarom is die file gedelete

Reply:
Ja, file verwijderd, maar kan dan toch weg uit je .gitignore?